packages/components-css/card-group/index.scss

```diff
@@ -0,0 +1,28 @@
+.frameless-card-group {
+  container-type: inline-size;
+  display: grid;
+  gap: 24px;
+  grid-auto-flow: dense;
+}
+
+@media (width >= 600px) {
+  .frameless-card-group {
+    display: grid;
+    gap: 24px;
+    grid-template-columns: 1fr 1fr;
+  }
+}
+
+@media (width <= 900px) {
+  .frameless-card-group {
+    display: grid;
+    gap: 24px;
+    grid-template-columns: 1fr;
+  }
+}
+
+@container (max-width: 400px) {
+  .frameless-card {
+    --utrecht-paragraph-font-size: var(--frameless-card-paragraph-font-size);
```

Comment:
Waarom zit deze font-size alleen in kleine containers? Voelt niet logisch?

Reply:
Zodat de font kleiner word voor mobile verse, maar ik heb hem nu in de container card content